client/js/components/procedure/SegmentsList/FilterFlyout.vue

```diff
@@ -399,6 +401,11 @@ export default {
                 this.$set(this.itemsObject, el.id, el)
               }
             })
+
+            // If the current filter is assignee, display amount of Segments that have assignee as null. That is given by the field missingResourcesSum
+            if (result.data[0].attributes.path === 'assignee') {
+              this.$set(this.itemsObject, "unassigned", { attributes: { count: result.data[0].attributes.missingResourcesSum , label: Translator.trans('not.assigned'), ungrouped : true,  selected: result.meta['unassigned_selected']} , id: "unassigned", type: "AggregationFilterItem", ungrouped : true})
+            }
```

Comment:
```suggestion
             if (result.data[0].attributes.path === 'assignee') {
              this.$set(this.itemsObject, 'unassigned', {
                attributes: {
                  count: result.data[0].attributes.missingResourcesSum,
                  label: Translator.trans('not.assigned'),
                  ungrouped: true,
                  selected: result.meta.unassigned_selected
                },
                id: 'unassigned',
                type: 'AggregationFilterItem',
                ungrouped: true
              })
            }
```
- Please add some line breaks for better readability
- Use single quotes instead of double ones
- You can also use dot notation for `result.meta.unassigned_selected`
- Delete spaces before comma